ui/mirage/factories/job-summary.js

```diff
@@ -9,7 +9,7 @@ export default Factory.extend({
   namespace: null,
 
   withSummary: trait({
```

Comment:
**question:**. If we're prioritizing edge use cases in our product roadmap, should we consider creating a mirage scenario called "Edge" where this scenario would live?

Currently it looks like this would impact all job types in every mirage scenario which could damage our mental model of Nomad, leading to architectural bugs later on.

Reply:
this is a good idea, long-term; for this particular feature, it might be a little hard: disconnected clients is testing an after-the-fact event (a client becomes disconnected). I'm not sure if there's such a thing as a timeout-based change in Mirage that would let us do this.